index.html

```diff
@@ -204,6 +204,18 @@ <h3 class="sub1">Multi-Post Stories</h3>
          </div>
       </div>
    </section>
+   <section class="contact">
+      <h2 class="contact-head">CONTACT US</h2>
+      <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <div class="form">
+         <form method="post" action="https://formspree.io/f/mknyrjkd" class="form">
+            <input type="text" name="name" id="name" placeholder="Text Input" required maxlength="30" class="input">
+            <input type="email" name="email" required class="input" value="jeremy@gmail.com">
+            <textarea placeholder="Enter Text Here" maxlength="500" class="textarea"></textarea>
+            <input type="submit" class="submit-btn" value="See Project">
+         </form>
+      </div>
+   </section>

```

Comment:
- [ ] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your design is missing the bottom social media icons and email addresses. Please place the social media icons and email address and make sure it matches the design.

See this image for the reference

![Screenshot (149)](https://user-images.githubusercontent.com/83803414/154421048-0cd4e1dc-a7f9-408b-9d2d-f52129492747.png)